lib/pbench/test/unit/agent/task/test_results_push.py

```diff
@@ -162,46 +149,105 @@ def test_multiple_meta_args_single_option():
+        runner = CliRunner(mix_stderr=False)
         result = runner.invoke(
             main,
             args=[
-                TestResultsPush.TOKN_SWITCH,
-                TestResultsPush.TOKN_TEXT,
-                TestResultsPush.ACCESS_SWITCH,
-                TestResultsPush.ACCESS_TEXT,
-                TestResultsPush.META_SWITCH,
-                TestResultsPush.META_TEXT_TEST,
-                TestResultsPush.META_SWITCH,
-                TestResultsPush.META_TEXT_FOO,
                 tarball,
+                TestResultsPush.ACCESS_SWITCH,
+                TestResultsPush.ACCESS_WRONG_TEXT,
             ],
         )
-        assert result.exit_code == 0, result.stderr
-        assert result.stdout == ""
+        assert result.exit_code == 2, result.stderr
+        assert "Error: Invalid value for '-a' / '--access': 'public/private'" in str(
+            result.stderr
+        )
 
     @staticmethod
     @responses.activate
     @pytest.mark.parametrize(
         "status_code,message,exit_code",
         (
-            (HTTPStatus.CREATED, None, 0),
-            (HTTPStatus.OK, {"message": "Dup"}, 0),
-            (HTTPStatus.OK, "Dup", 0),
-            (HTTPStatus.NO_CONTENT, {"message": "No content"}, 0),
-            (HTTPStatus.NO_CONTENT, "No content", 0),
+            (
+                HTTPStatus.CREATED,
+                None,
+                0,
+            ),
+            (
+                HTTPStatus.OK,
+                {"message": "Dup"},
+                0,
+            ),
+            (
+                HTTPStatus.OK,
+                "Dup",
+                0,
+            ),
+            (
+                HTTPStatus.NO_CONTENT,
+                {"message": "No content"},
+                0,
+            ),
+            (
+                HTTPStatus.NO_CONTENT,
+                "No content",
+                0,
+            ),
             (
                 HTTPStatus.REQUEST_ENTITY_TOO_LARGE,
                 {"message": "Request Entity Too Large"},
                 1,
             ),
-            (HTTPStatus.REQUEST_ENTITY_TOO_LARGE, "Request Entity Too Large", 1),
-            (HTTPStatus.NOT_FOUND, {"message": "Not Found"}, 1),
-            (HTTPStatus.NOT_FOUND, "Not Found", 1),
+            (
+                HTTPStatus.REQUEST_ENTITY_TOO_LARGE,
+                "Request Entity Too Large",
+                1,
+            ),
+            (
+                HTTPStatus.NOT_FOUND,
+                {"message": "Not Found"},
+                1,
+            ),
+            (
+                HTTPStatus.NOT_FOUND,
+                "Not Found",
+                1,
+            ),
+            (
+                0,
+                requests.exceptions.ConnectionError(
+                    "<urllib3.connection.HTTPConnection object at 0x1080854c0>: "
+                    "Failed to establish a new connection: [Errno 8] "
+                    "nodename nor servname provided, or not known"
+                ),
```

Comment:
I realize that this is basically a lift-and-shift, but unit tests don't _have_ to reflect reality...so I suspect that replacing this with
```python
                requests.exceptions.ConnectionError("Ooops")
```
would work fine.